Detections/ASimNetworkSessions/SeveralDenyActionsRegistered.yaml

```diff
@@ -0,0 +1,40 @@
+id: D66DD819-F2E6-43F7-8BC1-70BDE8011B39
+name: Several deny actions registered (AsimWebSession)
+description: |
+  'Identifies attack pattern when attacker tries to move, or scan, from resource to resource on the network and creates an incident when a source has more than 1 registered deny action in Azure Firewall.'
+tags:
+  - Id: f8dad4e9-3f19-4d70-ab7f-8f19ccd43a3e
+    version: 1.0.0
+  - Schema: ASIMNetworkSessions
+    SchemaVersion: 0.1.1
+severity: Medium
+requiredDataConnectors: []
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 1
+tactics:
+  - Discovery
+  - LateralMovement
+  - CommandAndControl
+relevantTechniques:
+  - T1046
+  - T1041
+  - T1071
+query: |
+  let threshold = 5;
+  imWebSession(dvcaction=dynamic(["Deny", "Drop", "Reset"]))
+  | summarize StartTime = min(TimeGenerated), Count=count() by SrcIpAddr, DstIpAddr, Url, DvcAction, NetworkApplicationProtocol
```

Comment:
You don't really want to group by device action. Different values are fine.

Reply:
alertDescriptionFormat: Potential attacker at address may have tried to move, or scan, from {{SrcIpAddr}} to {{DstIpAddr}} on the network using {{NetworkApplicationProtocol}} protocol and **failed more than 5 times**.

Description: Identifies attack pattern when attacker tries to move, or scan, from resource to resource on the network and creates an incident when a source failed more than 5 times to connect to the destination.